src/main/java/org/jabref/logic/journals/JournalInformationFetcher.java

```diff
@@ -0,0 +1,229 @@
+package org.jabref.logic.journals;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import javafx.util.Pair;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.WebFetcher;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.entry.identifier.ISSN;
+
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import kong.unirest.json.JSONArray;
+import kong.unirest.json.JSONException;
+import kong.unirest.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Fetches journal information from the JabRef Web API
+ */
+public class JournalInformationFetcher implements WebFetcher {
+    public static final String NAME = "Journal Information";
+    private static final Logger LOGGER = LoggerFactory.getLogger(JournalInformationFetcher.class);
+    private static final String API_URL = "https://jabref.org/api";
```

Comment:
@tobiasdiez We would like to separate the http representation from the api functionality. Can that also be hosted at `api.jabref.org`?

Reply:
Not really, both the website and in the future the office addin needs the `/api` endpoint, see https://github.com/JabRef/JabRefOnline/issues/576. I was reserving `api.jabref.org` for a potential endpoint for external services (e.g. if other people/apps want to call our api) but that requires more thought and work.